doc/lsst.pipe.base/creating-a-task.rst

```diff
@@ -0,0 +1,348 @@
+.. _creating-a-task:
```

Comment:
I get the feeling this and the following "Creating a command-line task" pages were copy-pasted from elsewhere and not written by you from scratch. The tone doesn't match the rest of the docs, they don't capitalize Butler everywhere, there is a "Documentation" section that simply says "to be written," and they feel a bit less accessible overall (for instance: "for now read existing tasks to see how it is done" re. how to use the Butler would be nicer as a not-yet-existing link to Butler documentation).

It's certainly better to have this content here than not, but these two pages would hugely benefit from feeling more like friendly tutorials and less like tired instruction manuals.